### ISSUE TITLE
2.5.22294: GameSelectorModel#gameUriExistsOnFileSystem:216 - java.net.URISyntaxException

### ISSUE THREAD

**Issue by tripleabuilderbot** [triplea-game/triplea#9222]
## Log Message
Illegal character in opaque part at index 2: C:\Users\Shadow Knight\triplea\downloadedMaps\iron_war\map\games\iron_war.xml

## TripleA Version
2.5.22294

## Java Version
11.0.6

## Operating System
Windows 10

## Memory
Heap utilization statistics [MB]
Used Memory: 20
Free memory: 1958
Total memory: 1979
Max memory: 1979

## Stack Trace
```
Exception: java.lang.IllegalArgumentExceptionIllegal character in opaque part at index 2: C:\Users\Shadow Knight\triplea\downloadedMaps\iron_war\map\games\iron_war.xml
java.lang.Exception
	at java.base/java.net.URI.create(URI.java:883)
	at games.strategy.engine.framework.startup.ui.panels.main.game.selector.GameSelectorModel.gameUriExistsOnFileSystem(GameSelectorModel.java:216)
	at java.base/java.util.Optional.filter(Optional.java:223)
	at games.strategy.engine.framework.startup.ui.panels.main.game.selector.GameSelectorModel.loadDefaultGameSameThread(GameSelectorModel.java:209)
	at games.strategy.engine.framework.GameRunner.showMainFrame(GameRunner.java:80)
	at java.base/java.lang.Thread.run(Thread.java:834)


Exception: java.net.URISyntaxExceptionIllegal character in opaque part at index 2: C:\Users\Shadow Knight\triplea\downloadedMaps\iron_war\map\games\iron_war.xml
java.lang.Exception
	at java.base/java.net.URI$Parser.fail(URI.java:2913)
	at java.base/java.net.URI$Parser.checkChars(URI.java:3084)
	at java.base/java.net.URI$Parser.parse(URI.java:3120)
	at java.base/java.net.URI.<init>(URI.java:600)
	at java.base/java.net.URI.create(URI.java:881)
	at games.strategy.engine.framework.startup.ui.panels.main.game.selector.GameSelectorModel.gameUriExistsOnFileSystem(GameSelectorModel.java:216)
	at java.base/java.util.Optional.filter(Optional.java:223)
	at games.strategy.engine.framework.startup.ui.panels.main.game.selector.GameSelectorModel.loadDefaultGameSameThread(GameSelectorModel.java:209)
	at games.strategy.engine.framework.GameRunner.showMainFrame(GameRunner.java:80)
	at java.base/java.lang.Thread.run(Thread.java:834)


```

**Comment by Keine-Andeutung** [comment]
I get this error message every time I start up TripleA. I started getting it in the current stable after I downloaded the then-current prerelease and opened a game of global 40 inside the prerelease. Accordingly, in my case, the path in the error message is pointing to the location of my global 40 map. Removing the global 40 map from the downloaded maps folder as well as removing all versions of TripleA and reinstalling them didn't remove the issue.
